src/index.js

```diff
@@ -167,12 +334,39 @@ const IN_CALL_BUTTONS_ARRAY = [
                 renderDisplay(undefined)
             }
 
+            // FAVOURITES_BUTTON_ELEMENT
+            favouritesButtonElement.onclick = () => {
+                renderFavourites()
```

Comment:
You have to tidy up naming your render functions - namely, I suggest you to append `Tab` to every  `render` function which renders tab